spec/lib/error_reporting_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+require 'error_reporting'
+
+RSpec.describe ErrorReporting do
+  describe '.call' do
+    let(:raven_double) { double(capture_message: :ok, capture_exception: :ok) }
+
+    before do
+      stub_const('Raven', raven_double)
+    end
+
+    let(:error) { Class.new(Exception).new("Nope.") }
+    let(:strng) { "Err, what about … no?" }
```

Comment:
Is this typo on purpose? Or is this nowadays already internetz-slang enough to no longer be considered a typo?

Reply:
haha, this wasn't a typo, but my way of having both statements aligned ;)